### DISCUSSION TITLE
Mark the day using a *nested* emoji tag

### DISCUSSION THREAD

**Discussion by ghost** [liamcain/obsidian-calendar-plugin#215] [Ideas]
I'd love to visually mark the days, but **I don't want a bunch of random emojis crowding my tag pane**. I think this applies for most people who actively use tags.

So is it possible to only use the nested emoji for visual tagging?
For example: 📅/🌍 would only mark the day with 🌍, and 📅/☎️ would only show ☎️.
This way all of these calendar-specific emojis would be hidden away and avid tag users would feel free to diversify them.

And if it IS possible, could you make it into a snippet? <3

**Comment by liamcain** [comment]
Was disappointed to see this doesn't already work, I think I had it working at one point but it got removed. I think this is something I can add pretty quickly. (And sorry for taking a while to get reply, it's been a busy month!!)

**Reply by ghost** [reply]
Oh that's great to hear. I'd very much appreciate it!
